packages/suite/src/hooks/wallet/coinmarket/form/useCoinmarketVerifyAccount.tsx

```diff
@@ -38,7 +40,7 @@ const getSelectAccountOptions = (
         });
 
         // have to be signed by private key
-        if (device?.connected) {
+        if (device?.connected && isNetworkSupported) {
             selectAccountOptions.push({ type: 'ADD_SUITE' });
         }
```

Comment:
Allowing to add a new account for the network only if the network is supported by Suite